docs/components/movement-sensor/imu/imu-wit-hwt905.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Configure a WitMotion HWT905-TTL IMU"
+linkTitle: "imu-wit-hwt905"
+weight: 10
+type: "docs"
+description: "Configure a WitMotion HWT905-TTL IMU on your machine. Once configured, use the API to obtain the AngularVelocity, Orientation, CompassHeading and LinearAcceleration."
+images: ["/icons/components/imu.svg"]
+# SMEs: Bucket Team
+---
+
+An [inertial measurement unit (IMU)](https://en.wikipedia.org/wiki/Inertial_measurement_unit) provides data for the [`AngularVelocity`](/components/movement-sensor/#getangularvelocity), [`Orientation`](/components/movement-sensor/#getorientation), [`CompassHeading`](/components/movement-sensor/#getcompassheading), [`LinearAcceleration`](/components/movement-sensor/#getlinearacceleration), and [`GetAccuracy`](/components/movement-sensor/#getaccuracy) methods.
+Acceleration and magnetometer data are available by using the [sensor](/components/sensor/) [`GetReadings`](/components/sensor/#getreadings) method, which IMUs wrap.
```

Comment:
Isn't it all movement sensors who wrap that?

Reply:
Well yes either way of wording it is still true. This is the way we word it on the other movement sensor pages--would you like me to change it to "which movement sensors wrap" on all of them?

Reply:
The way this is phrased, it sounds like `GetReadings` will give you acceleration and magnetometer stuff but not give you angular velocity or orientation. I had assumed `GetReadings` would give you all the readings the hardware can provide, but I haven't actually tried it out myself to double-check that.

Reply:
Ooh yeah I was keeping consistent with the other page but @penguinland do you think if it said “accelerometer” instead of “acceleration” that’d work?

Reply:
We should wire one of these up and try it out. I had expected it would give accelerometer, angular velocity, compass heading, and orientation. Now that I look at the code, I suspect it does _not_ give raw magnetometer data. but the real proof is in trying one out for real.

Reply:
![Screenshot_2024-05-14_17-14-27](https://github.com/viamrobotics/docs/assets/2861124/484e9dc4-f704-45e7-bc2e-0f9e5b699eaf)
![Screenshot_2024-05-14_17-14-11](https://github.com/viamrobotics/docs/assets/2861124/ce2f1b61-0604-43a0-8ec2-60fd421ff0d4)
Here are screenshots of what `GetReadings` and the main RC card look like. So, that's the data it can provide.

Reply:
Thanks so much for testing! So to me, it still seems like it'd be okay to say "accelerometer and magnetometer data" without getting overly specific. Or, do we like the following better?
```suggestion
To get orientation, angular velocity, linear acceleration and compass heading data, you can use the [sensor](/components/sensor/) [`GetReadings`](/components/sensor/#getreadings) method, which movement sensors inherit from the general sensor API.
```

Reply:
Either one LGTM!